### ISSUE TITLE
What are the criteria to understand about compliance?

### ISSUE THREAD

**Issue by etisdew** [w3c/wcag21#314]
I am trying to find my niche and build my resume. Found an adoption .org that is impossible to navigate in JAWS and they hadn't even heard of WCAG. I wish to volunteer my time and service while digesting and coming to lucid understanding of accessibility. Should they be considering WCAG compliance?

**Comment by etisdew** [comment]
The reason I post this as an issue is because I've found myself deaf blind and dumb in finding who is liable to maintain compliance, who enforces the guidelines, and to what extent.

**Comment by awkawk** [comment]
Jeff, yes they should be considering WCAG as the basis for the accessibility of their site as it is widely accepted as the most up to date accessibility standard.

I'm providing a quick response and closing this issue since this repository is for issues related to the development of the next version of WCAG - WCAG 2.1.

Good luck!

**Comment by etisdew** [comment]
Thank you very much, if you could use some assistance I hope to use this as my foot in the door by offering my services regarding accessibility, as someone who may require the use of TtS in the future. What are the best communication channels to get involved? I don't have a resume in the field and don't enjoy jumping through hoops to work for free but here I am.
